File: Data Structures & Algorithms/minimum-remove-to-make-valid-parentheses/submission-1.js
class Solution {
    /**
     * @param {string} s
     * @return {string}
     */
    minRemoveToMakeValid(s) {
        let count = 0;
        let stack = [];

        for (let i of s) {
            if (i === ")" && count > 0) {
                stack.push(i);
                count -= 1;
            } else if (i === "(") {
                stack.push(i);
                count += 1;
            } else if (i !== ")") {
                stack.push(i);
            }
        }

        let res = [];

        for (let i = stack.length - 1; i >= 0; i--) {
            if (stack[i] === "(" && count > 0) {
                count -= 1;
            } else {
                res.push(stack[i]);
            }
        }

        res.reverse();

        return res.join("");
    }
}

File: Data Structures & Algorithms/two-integer-sum-ii/submission-1.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        l, r = 0, len(numbers) - 1
        res = numbers[l] + numbers[r]

        while (res != target):
            if (res < target):
                l += 1
                res = numbers[l] + numbers[r]
            elif (res > target):
                r -= 1
                res = numbers[l] + numbers[r]

        return [l+1, r+1]